includes/API/Coupons_Controller.php

```
@@ -0,0 +1,262 @@
+	 * @param WP_REST_Request $request Full details about the request.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function wcpos_get_all_posts( $request ) {
+		global $wpdb;
+
+		// Start timing execution.
+		$start_time = microtime( true );
+
+		$modified_after        = $request->get_param( 'modified_after' );
+		$dates_are_gmt         = true; // Dates are always in GMT.
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$dates_are_gmt'. (undefined)

(UnusedLocalVariable)